v-next/hardhat-mocha-test-runner/src/hookHandlers/config.ts

```diff
@@ -72,10 +74,17 @@ const mochaConfigType = z.object({
 
 const userConfigType = z.object({
   mocha: z.optional(mochaConfigType),
-  test: unionType(
```

Comment:
was the original config path a bug? I noticed you changed it from `config.test` to `config.paths.test`, which makes sense, but I’m wondering why it was like that initially. Did we miss updating it in a previous PR?

Reply:
Oh, it was a bug, and the fix went unnoticed in this PR. I was meaning to send a separate one.